src/Agent/NewRelic/Agent/Core/DistributedTracing/Samplers/AlwaysOnSampler.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace NewRelic.Agent.Core.DistributedTracing.Samplers;
+
+public class AlwaysOnSampler : ISampler
+{
+    public ISamplingResult ShouldSample(ISamplingParameters samplingParameters) => new SamplingResult(true, 2.0f);
```

Comment:
We should be able to apply the same optimization that I'm suggesting for the AlwaysOff sampler here too.